graph/patterns/default-properties.md

```diff
@@ -0,0 +1,201 @@
+# Default properties
+
+Microsoft Graph API Design Pattern
+
+*The default properties pattern allows API producers to omit specific properties from the response unless they are explicitly requested using `$select`. The pattern also uses an instance annotation to inform callers that other properties are also available. The same annotation is also use to encourage callers to use `$select`.*
+
+## Problem
+
+API producers want to control the set of properties that their entities return by default, when the incoming request does not specify a `$select`. This can be desirable when an entity type has many properties or an API producer needs to add properties that are computationally expensive to return by default.
+
+## Solution
+
+For incoming requests targeting an entity type where the caller does not specify a `$select`` clause, API producers **may** return a subset of the entity type's properties, omitting computationally expensive properties.
+
+To get the non-default properties of an entity type, callers must explicitly request them using `$select`.
+
+## When to use this pattern
+
+API producers should use this pattern when adding expensive or non-performant properties to an existing entity type, or when adding properties to an entity type that has already grown too large (with more than 20 properties).
+
+## Issues and considerations
+
+- Do **not** rely on the `ags:Default` schema annotation for default properties functionality, as this is a legacy implementation. Returning default properties **must** be implemented by API producers.
+- Changing a default property to non-default is considered a breaking change.
+- One of the challenges with default properties is informing developers that the response does not contain the full set of properties. To solve for this discovery problem, if the response contains default properties only, then:
+  - the response **must** contain a `@microsoft.graph.tips` instance annotation.
+  - the `@microsoft.graph.tips` instance annotation **must** only be emitted if the client uses "developer mode" via the `Prefer: ms-graph-dev-mode` HTTP request header. It is expected that this header will only be used by client developer and scripting tools like Graph Explorer, the Microsoft Graph Postman collections, and Microsoft Graph PowerShell.
```

Comment:
PowerShell?

Reply:
Do my eye's deceive me, or do I not mention PowerShell at the end of the sentence?  If that's not what your comment is about, please add more detail...